spec/features/requests/request_spec.rb

```diff
@@ -1596,6 +1596,120 @@
         end
       end
     end
+
+    context 'An Alma user' do
+      let(:alma_login_response) { fixture('/alma_login_response.json') }
+      let(:user) { FactoryBot.create(:valid_alma_patron) }
+      before do
+        stub_request(:get, "#{Alma.configuration.region}/almaws/v1/users/#{user.uid}?expand=fees,requests,loans")
+          .to_return(status: 200, headers: { "Content-Type" => ["application/json", "charset=UTF-8"] },
+                     body: alma_login_response)
+        login_as user
+      end
+
+      it "does not allow physical pickup request On Order SCSB Recap Item" do
+        stub_scsb_availability(bib_id: "9994933183506421", institution_id: "PUL", barcode: '33333059902417')
+        visit 'requests/SCSB-6710959'
+        expect(page).not_to have_content 'Electronic Delivery'
+        expect(page).not_to have_content 'Physical Item Delivery'
+        expect(page).to have_content 'Request options for this item are currently only available to Faculty, Staff, and Students.'
+      end
+
+      it "allows a physical pickup request of ReCAP Item" do
+        stub_scsb_availability(bib_id: "9941151723506421", institution_id: "PUL", barcode: '32101050751989')
+        visit 'requests/9941151723506421?mfhd=22492702000006421'
+        expect(page).not_to have_content 'Electronic Delivery'
+        expect(page).to have_content 'Physical Item Delivery'
+      end
+
+      it "allows only physical pickup to enumerated annex item" do
```

Comment:
@carolyncole is this test specific to annex or to annex enumerated item? Will it allow physical pickup if it is annex item and not enumerated?

Reply:
@christinach this item is enumerated so I put it in the description.  The code should allow access to annex items either enumerated or not.